README.md

```diff
@@ -180,6 +180,8 @@ following properties:
   on the state node. If the value is an object, its content will
   also be interpreted to create children nodes.
 
+An empty array remove the root node (privided with `id`) and it's childrens.
```

Comment:
@SL Rephrase: `Specifying an empty array removes the specified state node and all of its children.`